Gemfile

```diff
@@ -2,8 +2,7 @@ source "https://rubygems.org"
 
 gem "fastlane"
 
-plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
-eval_gemfile(plugins_path) if File.exist?(plugins_path)
```

Comment:
Hmm this was the recommended way to add the Pluginfile by fastlane. I guess by hardcoding the path, it might not work from a different folder... But seems like dependabot doesn't work with this, so I think we can go with this for now and see if there are any issues.

Reply:
I saw they did it in https://github.com/getsentry/sentry-cocoa/pull/982/files too which gave me confidence lol